static/.eslintrc.cjs

```diff
@@ -14,6 +14,8 @@ module.exports = {
     "vue/attributes-order": "error",
     "vue/order-in-components": "error",
     "vue/require-explicit-emits": "error",
+    "vue/attribute-hyphenation": "error",
+    "cypress/no-async-tests": "off",
```

Comment:
Nur zum Verständnis: Was genau verändern die beiden Regeln/warum brauchen wir die?

Reply:
Gut dass du nachfragst :)

Die erste Regel sorgt dafür dass man im html kebap-case statt camelCase verwenden muss, War vorher warning, hätte also gut passieren können dass wir das mal übersehen, daher jetzt als verpflichtend eingestellt.
Den anderen Lint zu deaktivieren, erlaubt uns async/await in cypress tests zu nutzen - dadurch können wir einfach `await login(); ...` schreiben (statt `login().then(() => { ... })`)